migration_toolkit_for_containers/mtc-release-notes.adoc

```diff
@@ -8,7 +8,7 @@ toc::[]
 
 The release notes for {mtc-full} ({mtc-short}) describe new features and enhancements, deprecated features, and known issues.
 
-The {mtc-short}) enables you to migrate application workloads between {product-title} clusters at the granularity of a namespace.
+The {mtc-short} enables you to migrate application workloads between {product-title} clusters at the granularity of a namespace.
```

Comment:
Check the word usage statement for 'migrate' to ensure that is the best word choice here. Being closer to this content, you can determine whether 'migrate' is the best choice.
https://www.ibm.com/docs/en/ibm-style?topic=word-usage

Reply:
@tmalove Yes, "migrate" is the correct word. I was just cleaning up an extra ) here.  Thanks.